winterize/init.lua

```diff
@@ -0,0 +1,81 @@
+local snowy_dirt_tiles = minetest.registered_nodes["default:dirt_with_snow"].tiles
+local snowy_dirt_sounds = minetest.registered_nodes["default:dirt_with_snow"].sounds
+local grasses = {"dry_grass", "grass", "coniferous_litter",}
+local leaves = {"leaves", "aspen_leaves", "jungleleaves"}
+
+minetest.register_node("winterize:ice", { -- breaks instantly, drops nothing
+	drawtype = "nodebox",
+	description = "Ice",
+	tiles = {"winterize_ice_seethrough.png"},
+	buildable_to = true,
+	floodable = true,
+	paramtype = "light",
+	sunlight_propogates = true,
+	node_box = {
+		type = "fixed",
+		fixed = {
+			{-0.5, -0.5, -0.5, 0.5, -0.4, 0.5},
+		},
+	},
+	collision_box = {
+		type = "fixed",
+		fixed = {
+			{-0.5, -0.5, -0.5, 0.5, -0.4, 0.5},
+		},
+	},
+	drop = "",
+	groups = {dig_immediate = 3, cools_lava = 1, slippery = 4},
```

Comment:
```suggestion
	groups = {dig_immediate = 3, slippery = 4},
```

`cools_lava` is not necessary as the lava cooling ABM has been removed.

Reply:
This was copied straight from `default`. Will fix